.github/workflows/unit-test-main.yml

```diff
@@ -13,6 +13,6 @@ jobs:
 
       - name: Run tests 🧪
         uses: ./.github/actions/unit-tests-action
-        with:
-          supabaseUrl: ${{ secrets.VITE_SUPABASE_URL }}
-          supabaseKey: ${{ secrets.VITE_SUPABASE_KEY }}
+        env:
```

Comment:
> As [GitHub documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepswith) states, supabaseUrl is transformed to INPUT_SUPABASEURL which is not what we want in the app.

thoughts: I don't think that's a point of concern because we explicitly give correct names to env vars [here](https://github.com/aabounegm/cast/blob/feature/fsd-entities-2/.github/actions/unit-tests-action/action.yml#L40-L41). It was working before, as far as I'm aware.

question: what exactly is the reason why you changed it in this way?

Reply:
Oh wait, I didn't even notice the `actions` folder. Ok, in this case I have no idea how to fix it.
You can try running this PR locally – it works. But not in the CI environment – `constants.SUPABASE_URL` is an empty string.

Reply:
Ok, found. In the [`actions/unit-tests-action/action.yml`](https://github.com/aabounegm/cast/blob/feature/fsd-entities-2/.github/actions/unit-tests-action/action.yml#L40-L41) file, there were no Environmental variables next to the Unit testing job, only in the E2E tests